Subject: [PATCH] pocket1 Use Back Button As Down on Pocket1
 * on the pocket1, the volume down buttons is unusually registered as the
 back button. In Android, this is worked around by changing it in the kl
 file. So modify Lineage recovery make the back button do the same as the
 volume down button.

diff --git a/device.cpp b/device.cpp
index 067a0082..ba3a0e4e 100644
--- a/device.cpp
+++ b/device.cpp
@@ -152,6 +152,7 @@ int Device::HandleMenuKey(int key, bool visible) {
     case KEY_MENU:
     case BTN_NORTH:
     case BTN_DPAD_DOWN:
+    case KEY_BACK:
       return kHighlightDown;
 
     case KEY_LEFTSHIFT:
@@ -180,7 +181,6 @@ int Device::HandleMenuKey(int key, bool visible) {
       return kGoHome;
 
     case KEY_BACKSPACE:
-    case KEY_BACK:
       return kGoBack;
 
     case KEY_REFRESH:

